tests/components/climate/test_demo.py

```diff
@@ -237,6 +237,13 @@ def test_set_aux_heat_bad_attr(self):
         self.hass.block_till_done()
         self.assertEqual('off', state.attributes.get('aux_heat'))
 
+    def test_set_aux_heat_on_ecobee(self):
```

Comment:
@ballob: Can you please clarify the use of the ecobee demo tests. They definitely get run, so they seem to be useful. In addition, if tests are not provided, coverage goes down and the CI check will not pass.